src/context/ChainApiContext.tsx

```diff
@@ -114,18 +116,18 @@ type ChainApiContext = {
   reconnect: () => void;
 };
 
-export function useApiReconnect() {
-  const context = useContext(ChainApiContext);
+// export function useApiReconnect() {
```

Comment:
In case you think we need to test something before cleaning this up, let's create an issue to clean this so we don't forget. You can add a todo with the link to the issue here

Reply:
Cleaned it.